src/sympc/tensor/replicatedshare_tensor.py

```diff
@@ -581,6 +581,43 @@ def reconstruct(
 
         raise ValueError("Invalid security Type")
 
+    @staticmethod
+    def distribute_shares_to_party(
+        shares: List[Union[ShareTensor, torch.Tensor]],
+        party_rank: int,
+        session: Session,
+    ):
+        """Distributes shares to party.
+
+        Args:
+            shares (List[Union[ShareTensor,torch.Tensor]]): Shares to be distributed.
+            party_rank (int): Rank of party.
+            session (Session): Current session
+
+        Returns:
+            tensor (ReplicatedSharedTensor): Tensor with shares
+        """
+        party = session.parties[party_rank]
+        nshares = session.nr_parties - 1
```

Comment:
I'd rather not introduce this variable which doesn't seem very useful

Reply:
I feel it makes the code more easily read rather than having just session.nr_parties-1 somewhere, someone reading the code would easily understand its the number of sharea.